### PR TITLE
Allow disabling the spacebar remap

### PR DESCRIPTION
**Pull request by LoicAG**
This takes care of #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ Allow vim-terraform to override your `.vimrc` indentation syntax for matching fi
 
     let g:terraform_align=1
 
+Allow vim-terraform to re-map the spacebar to fold / unfold. Defaults to 1 which is on.
```
Comment (pgporada) [inline README.md]: Change this to default to off and I think it should be ok.

Comment (LoicAG) [inline README.md]: OK, fine by me, the whole point is for me to be able to disable it.

**Comment by pgporada** [comment]
Sure, let's default the functionality to off though and people can toggle
it on at their leisure. The unfolding/folding might be a surprise to some
who have never seen it.

**Comment by LoicAG** [comment]
OK, I had initially left it on by default so as not to surprise current users who rely on it

**Comment by pgporada** [comment]
Hmm, I see your point. ¯\_(ツ)_/¯

**Comment by pgporada** [comment]
Thank you for the help!

**Comment by LoicAG** [comment]
Although if they're like me, they might actually be happy to recover normal functionality of their space bar :)

**Comment by LoicAG** [comment]
Thanks for the cool plugin :)

**Pull request merged**